### PR TITLE
add "message sent" success message

### LINKED ISSUE #2638: Email Dataset Contact: Missing success message that email was sent
Their is no "success" notification when someone uses the "email icon" contact in a dataset. The page refreshes leaving the user wondering if an email was sent.

### PR DESCRIPTION
**Pull request by pdurbin**
**What this PR does / why we need it**:

After submitting a message to a dataset contact, you should see "message sent".

**Which issue(s) this PR closes**:

- Closes #2638

**Special notes for your reviewer**:

I'm happy to fix the indentation above the line of Java I added.

**Suggestions on how to test this**:

Use the "Contact Owner" button on a dataset.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

You now see "message sent":

Screenshot 2023-12-13 at 2 14 42 PM

In addition, there is a new placeholder called name@email.xyz when you use the contact form:

Screenshot 2024-01-08 at 10 30 11 AM
Screenshot 2024-01-08 at 10 30 03 AM

**Is there a release notes update needed for this change?**:

I don't think so.

**Additional documentation**:

None.

### CODE REVIEW

**Comment by jp-tosca** [comment]
I totally agree with you with what you wrote on https://github.com/IQSS/dataverse/issues/2638#issuecomment-255737685 : 

> Also, I was tripped up that "From" was asking for my email address rather than my name:

We could use a placeholder just to make it more clear for people that they need to input an email address from the begining, what do you think?

image

File: src/main/java/propertyFiles/Bundle.properties [hunk 1]
```diff
@@ -155,6 +155,7 @@ contact.support=Support
 contact.from=From
 contact.from.required=User email is required.
 contact.from.invalid=Email is invalid.
+contact.from.emailPlaceholder=valid@email.org
```
Comment (pdurbin) [inline src/main/java/propertyFiles/Bundle.properties:158]: @jp-tosca for consistency should we use name@email.xyz?

src/main/java/propertyFiles/citation.properties
175:datasetfieldtype.datasetContactEmail.watermark=name@email.xyz

Screenshot 2024-01-05 at 11 30 33 AM

Comment (pdurbin) [reply src/main/java/propertyFiles/Bundle.properties:158]: @jp-tosca and I talked about this. I'll make the emails match.

Comment (pdurbin) [reply src/main/java/propertyFiles/Bundle.properties]: Added in dbab6ca.

**Comment by pdurbin** [comment]
@jp-tosca as discussed, I updated the email to match. I also put screenshots in the PR description and merged the latest from develop.

**Review by jp-tosca — APPROVED** [review]

**Pull request merged**